mods/technic-master/technic/machines/LV/cnc_nodes.lua

```diff
@@ -21,12 +21,31 @@ technic.cnc.register_all("default:stone",
                 {cracky=3, not_in_creative_inventory=1},
                 {"default_stone.png"},
                 S("Stone"))
+-- STONE BRICK - added
+---------------
+technic.cnc.register_all("default:stonebrick",
+                {cracky=3, not_in_creative_inventory=1},
+                {"default_stone.png"},
```

Comment:
you have to provide the same texture you want from the cnc output, `default_stone_brick.png` in this case (https://github.com/minetest/minetest_game/blob/1ade8941b497e71dd05e80827e5f34a4659e47b0/mods/default/nodes.lua#L271)